helmfile.d/helmfile-60.teams.yaml

```diff
@@ -130,7 +130,7 @@ releases:
           - low_priority_channel: {{ $team | get "msteams.lowPrio" }}
     {{- end }}
   - name: grafana-dashboards-{{ $teamId }}
-    installed: true
+    installed: {{ $team.monitoringStack.enabled }}
```

Comment:
The `$team.monitoringStack.enabled` needs a default value.
Use get `installed: {{ $team | get "monitoringStack.enabled" true }}`

Reply:
adjusted